lib/model/query/entities.js

```diff
@@ -501,8 +520,25 @@ const _processSubmissionEvent = (event, parentEvent) => async ({ Audits, Dataset
         throw (err);
       }
     }
-  else if (entityData.system.create === '1' || entityData.system.create === 'true')
-    maybeEntity = await Entities._createEntity(dataset, entityData, submissionId, submissionDef, submissionDefId, event, parentEvent, forceOutOfOrderProcessing);
+  else if (entityData.system.create === '1' || entityData.system.create === 'true') {
+    try {
+      maybeEntity = await Entities._createEntity(dataset, entityData, submissionId, submissionDef, submissionDefId, event, parentEvent, forceOutOfOrderProcessing);
+    } catch (err) {
+      // There was a problem creating the entity
+      // If it is a uuid collision, check if the entity was created via an update
+      // in which case its ok to apply this create as an update
+      if (err.problemCode === 409.3) {
+        const rootDef = await Entities.getDef(dataset.id, entityData.system.id, new QueryOptions({ root: true })).then(o => o.orNull());
+        if (rootDef && rootDef.aux.source.forceProcessed) {
+          maybeEntity = await Entities._updateEntity(dataset, entityData, submissionId, submissionDef, submissionDefId, event, forceOutOfOrderProcessing, true);
```

Comment:
I'm sure this is unlikely, but this logic does leave open the possibility that two separate entity-create submissions could update the same entity, assuming the entity was created from a force-processed offline update, and the two entity-create submissions specify the same UUID. In general, if an entity was created from an entity-create submission, that entity will only ever be associated with a single entity-create submission (the first submission). However, here, if an entity was created from a force-processed offline update, it can end up being associated with an arbitrary number of entity-create submissions, all applied as updates. I don't think we necessarily need to address that in this PR though (I could file an issue about it).

Reply:
I think this is and an edge case of an edge case. You'd have to be creating entities (via different submissions) with the same uuids as an entity you were also updating, all in a situation where every one of these creates got dropped and had an update force-applied first. I don't think this would happen in practice unless someone was intentionally messing with the system.

Reply:
I think it's a little more likely than that, but still unlikely. It could happen if an entity was created from a force-processed offline update, then someone sent the entity-create submission multiple times (maybe they'd have to adjust the submission instance ID each time).

> unless someone was intentionally messing with the system.

In general, I think someone intentionally messing with the system is a real possibility that we should guard against. Not all data collectors are highly trusted, especially for public links. In this case though, it doesn't like a big problem for someone to be able to apply multiple entity-create submissions as updates to the same entity.

I think you're saying that we shouldn't file an issue about this case, right? I'm fine with that, we can always address it if it comes up.

Reply:
I don't think we should file an issue about it